src/main/bindings/c/bindings.h

```diff
@@ -662,7 +662,11 @@ SysCallReturn rustsyscallhandler_fcntl(SysCallHandler *sys, const SysCallArgs *a
 
 SysCallReturn rustsyscallhandler_fcntl64(SysCallHandler *sys, const SysCallArgs *args);
 
-SysCallReturn log_syscall(Process *proc, const char *name, const char *args, SysCallReturn result);
+SysCallReturn log_syscall(Process *proc,
+                          pid_t tid,
```

Comment:
I'd lean towards `host::thread::ThreadId`  to keep it distinct from native thread id's.

(Heh, initially I hadn't backtick quoted, and gitlab emojified to host::thread::ThreadId)

Reply:
I'm not sure what you mean here. Do you mean to export `host::thread::ThreadId` as a C type and create a new `ThreadId` object from C?

Reply:
Oops, yeah I missed that this was a C API. Nevermind :)

> (And I wonder if Rust supports emojis in module names, then we could replace the thread module with :thread:)

:rofl: